src/main/java/games/dicemonastery/DiceMonasteryTurnOrder.java

```diff
@@ -18,14 +18,15 @@
 public class DiceMonasteryTurnOrder extends TurnOrder {
 
     public DiceMonasteryTurnOrder(int nPlayers, DiceMonasteryParams params) {
-        super(nPlayers, params.YEARS);
+        super(nPlayers, params.YEARS * 4 - 1);
```

Comment:
I spot hard-coded numbers that I don't like

Reply:
Oh alright then - I suppose I can add a parameter to say how many seasons there are in a year....just in case planetary cosmology changes.

Reply:
It can also be a comment to say what the 4 is :)